dao/src/main/java/com/epam/esm/dao/entity/GiftCertificateEntity.java

```diff
@@ -0,0 +1,106 @@
+package com.epam.esm.dao.entity;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.Objects;
+
+public class GiftCertificateEntity implements Serializable {
+
+  private int id;
+  private String name;
+  private String description;
+  private float price;
+  private int duration;
+  private Timestamp createDate;
+  private Timestamp lastUpdateDate;
+
+  public GiftCertificateEntity() {
+  }
+
+  public int getId() {
+    return id;
+  }
+
+  public void setId(int id) {
+    this.id = id;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public String getDescription() {
+    return description;
+  }
+
+  public void setDescription(String description) {
+    this.description = description;
+  }
+
+  public float getPrice() {
+    return price;
+  }
+
+  public void setPrice(float price) {
+    this.price = price;
+  }
+
+  public int getDuration() {
+    return duration;
+  }
+
+  public void setDuration(int duration) {
+    this.duration = duration;
+  }
+
+  public Date getCreateDate() {
+    return createDate;
+  }
+
+  public void setCreateDate(Timestamp createDate) {
+    this.createDate = createDate;
+  }
```

Comment:
why do you use Timestamp in setter and Date in getter for the same field?

Reply:
My mistake, thank you